libraries/ride-fuzzer-lib/src/main/java/com/adobe/ride/libraries/fuzzer/MetadataFuzzer.java

```diff
@@ -29,23 +37,23 @@
  * 
  */
 public class MetadataFuzzer {
-  protected ModelObject entity;
+  public ModelObject entity;
   protected JSONObject properties;
-  protected JSONObject instance;
-  protected String serviceName;
-  protected Method requestMethod;
-  protected String contentType;
-  protected RequestSpecBuilder requestBuilder;
-  protected Filter[] filters;
+  public String serviceName;
+  public Method requestMethod;
+  public String contentType;
+  public RequestSpecBuilder requestBuilder;
+  public Filter[] filters;
+  protected String root;
```

Comment:
For optimizations to the Fuzzing flow, these properties need to be available to the MetadataEngine class.